package.json

```diff
@@ -39,15 +39,16 @@
     "fix:tslint": "tslint --fix --project .",
     "test": "run-s build test:*",
     "test--disabled--:lint": "tslint --project . && prettier \"src/**/*.ts\" --list-different",
-    "test:unit": "mocha -r ts-node/register src/**/*.spec.ts",
-    "coverage": "nyc mocha -r ts-node/register src/**/*.spec.ts",
+    "test:unit": "mocha -r ts-node/register src/**/*.spec.ts --exit",
+    "test:integration": "mocha -r ts-node/register test/integration/**/*.ts test/integration/*.ts --exit",
```

Comment:
Is there a reason we need both? `test/integration/**/*.ts test/integration/*.ts`